cypress/support/utils.ts

```diff
@@ -112,17 +158,20 @@ const setTinyMceContent = (tinyMceId, content) => {
   });
 };
 
-const getTinyMceContent = (tinyMceId) => {
+const getTinyMceContent = (tinyMceId: string) => {
   cy.get(`#${tinyMceId}`).should('exist');
 
-  cy.window().then((win) => {
+  return cy.window().then((win) => {
     const editor = win.tinyMCE.editors[tinyMceId];
 
     return editor.getContent();
   });
 };
 
-const testActionButton = (title, state) => {
+const testActionButton = (
+  title: string,
+  state: 'completed' | 'active' | 'inactive' | 'neutral' | 'invisible'
```

Comment:
I wonder if this can be just of the type `ActionButtonState` which has the values